src/ansible_navigator/configuration_subsystem/navigator_configuration.py

```diff
@@ -475,6 +475,17 @@ class Internals(SimpleNamespace):
             subcommands=["doc"],
             value=SettingsEntryValue(default="module"),
         ),
+        SettingsEntry(
+            name="pull_arguments",
+            cli_parameters=CliParameters(action="append", nargs="+", short="--pa"),
+            settings_file_path_override="execution-environment.pull.arguments",
+            short_description=(
+                "Specify any additional parameters that should be added to the"
+                " pull command when pulling an execution environment from a container"
+                " registry. e.g. --pa \"'--tls-verify=false'\""
```

Comment:
Are the doubled up quotes needed here? This looks odd.

Reply:
researching

Reply:
ok............  this has to do with it being recognized as a unknown arg because it has a `--` in it....

```
import argparse
import shlex

parser = argparse.ArgumentParser(exit_on_error=False)
parser.add_argument("--pass-through", nargs="+")

# No quotes
command = "--pass-through --other=5"
try:
    known, unknown = parser.parse_known_args(shlex.split(command))
except argparse.ArgumentError as exc:
    assert "argument --pass-through: expected at least one argument" in str(exc)

# single quotes
command = "--pass-through '--other=5'"
try:
    known, unknown = parser.parse_known_args(shlex.split(command))
except argparse.ArgumentError as exc:
    assert "argument --pass-through: expected at least one argument" in str(exc)

# double quotes
command = '--pass-through "--other=5"'
try:
    known, unknown = parser.parse_known_args(shlex.split(command))
except argparse.ArgumentError as exc:
    assert "argument --pass-through: expected at least one argument" in str(exc)

# nested quotes
command = "--pass-through \"'--other=5'\""
known, unknown = parser.parse_known_args(shlex.split(command))
assert known.pass_through == ["'--other=5'"]

# equal raw
command = "--pass-through=--other=5"
known, unknown = parser.parse_known_args(shlex.split(command))
assert known.pass_through == ["--other=5"]

# equal quoted
command = "--pass-through='--other=5'"
known, unknown = parser.parse_known_args(shlex.split(command))
assert known.pass_through == ["--other=5"]
```